src/tty/windows.rs

```diff
@@ -228,14 +198,16 @@ impl RawReader for ConsoleRawReader {
                 };
                 let c = rc?;
                 if meta {
-                    return Ok(KeyPress::Meta(c));
+                    return Ok(KeyPress::meta(c));
                 } else {
                     let mut key = keys::char_to_key_press(c);
-                    if key == KeyPress::Tab && shift {
-                        key = KeyPress::BackTab;
-                    } else if key == KeyPress::Char(' ') && ctrl {
-                        key = KeyPress::Ctrl(' ');
+                    if key == KeyPress::TAB && shift {
+                        key = KeyPress::BACK_TAB;
+                    } else if key == KeyPress::from(' ') && ctrl {
+                        key = KeyPress::ctrl(' ');
                     }
+                    // XXX should this be key.with_mods(mods)? Leaving it as-is
+                    // for now because it seems deliberate.
```

Comment:
We should do something like https://github.com/crossterm-rs/crossterm/blob/master/src/event/sys/windows/parse.rs#L120.
To support something like Ctrl-Alt-A.